circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
+        parallel: true
+        files:
+          - client/**/*.js
+          - client/**/*.jsx
+          - server/**/*.js
+          - server/**/*.jsx
+    - NODE_ENV=development make build && NODE_ENV=development ./bin/run-tests:
```

Comment:
Yup, complete agreement. I was debating doing that here, but copying the dev config was simpler and less prone to introduce crazy things.